packages/middleware-ssec/src/index.ts

```diff
@@ -78,3 +77,19 @@ export const getSsecPlugin = (config: PreviouslyResolved): Pluggable<any, any> =
     clientStack.add(ssecMiddleware(config), ssecMiddlewareOptions);
   },
 });
+
+function isValidBase64(str: string) {
+  const base64Regex = /^(?:[A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/;
+
+  if (!base64Regex.test(str)) return false;
+
+  try {
+    const decodedBytes = Buffer.from(str, "base64");
```

Comment:
encoding/decoding must use the SerdeContext's encoder/decoders, i.e. `options` above. 

Buffer is not available in browsers.